docker/configs/redis.conf

```diff
@@ -0,0 +1,1335 @@
+###############################################################################
+#
+# HyperSciecne notes:
+#  * this file is copied from ubuntu's redis-server package
+#  * bind is changed to listen only on IPv4
+#  * port is changed to 22122 (this one is used in dynomite setup)
+#  * log configuration is disabled, otherwise container size will grow
+#  * It is important to keep at least one "save" configuration - when
+#    there's a save configuration redis will do a save on SIGTERM signal
+#  * tcp-backlog 511 is commented because of warning about docker host
+#    /proc/sys/net/core/somaxconn
+#    the warning is still there....
```

Comment:
What's the warning?

Feels like it should be a non-issue, but you might want to set `somaxconn` for the docker image to be higher than/equal to the value set for `tcp-backlog` here.